scripts/mutect2_wdl/mutect2_multi_sample.wdl

```diff
@@ -1,68 +1,34 @@
 #  Run Mutect 2 on a list of tumors or tumor-normal pairs
 #
 #  Description of inputs
-#  gatk4_jar: java jar file containing gatk 4 (protected)
+#  gatk: java jar file containing gatk 4 (protected)
```

Comment:
This comment seems to be out of date.  Maybe you can delete it even?

Reply:
done